foundations/foundation-metrics/src/main/java/io/servicecomb/foundation/metrics/MetricsServoRegistry.java

```diff
@@ -442,21 +417,21 @@ public String call() {
         int operLatency = instance.getExecutionTimeMean();
         long totalCallCount = successCount + failureCount;
         cumulativeTotalCount += totalCallCount;
-        int windowTime = instance.getProperties().metricsRollingStatisticalWindowInMilliseconds().get() / 1000;
+        double windowTime = (double)instance.getProperties().metricsRollingStatisticalWindowInMilliseconds().get() / (double)1000;
         double qpsVal = (double) (totalCallCount) / windowTime;
         BigDecimal bigDecimal = new BigDecimal(qpsVal);
         BigDecimal bigDecimalVal = bigDecimal.setScale(1, RoundingMode.HALF_DOWN);
         Double tpsOper = bigDecimalVal.doubleValue();
-        tpsAndLatencyMap.put("TPS-" + instance.getCommandKey().name(), String.valueOf(tpsOper));
-        tpsAndLatencyMap.put("Latency-" + instance.getCommandKey().name(), String.valueOf(operLatency));
+        //tpsAndLatencyMap.put("TPS-" + instance.getCommandKey().name(), String.valueOf(tpsOper));
+        //tpsAndLatencyMap.put("Latency-" + instance.getCommandKey().name(), String.valueOf(operLatency));
         insTotalTps += tpsOper;
         insTotalLatency += operLatency;
       }
 
-      double instanceLatency = (double) (insTotalLatency) / cumulativeTotalCount;
+      double instanceLatency = insTotalLatency / cumulativeTotalCount;
 
-      tpsAndLatencyMap.put("TPS Instance_Level", String.valueOf(insTotalTps));
-      tpsAndLatencyMap.put("Latency Instance_Level", String.valueOf(instanceLatency));
+      tpsAndLatencyMap.put("tps", String.valueOf(insTotalTps));
+      tpsAndLatencyMap.put("latency", String.valueOf(instanceLatency));

```

Comment:
what's the length of latency string?

Reply:
sorry what's means

Reply:
add "servicecomb.metrics.round_places“ setting set round places ,default is 1